frontend/src/assets/content/arp/rules-to-know/rules.md

```diff
@@ -4,25 +4,25 @@ The below applies to all available Christmas tree cutting areas in the {{forestN
 
 * The tree stump must be less than {{stumpDiameter}} inches in diameter.
 * You must leave a stump height of {{stumpHeight}} inches or less.
-* Only cut trees within the National Forest boundary.
-* Do not cut any tree designated with paint. 
+* Only cut trees within the national forest boundary.
 * Tree-topping is prohibited. Cut down the whole tree. 
-* After cutting your tree, take or scatter any unused portion of your tree.
+* Cut the leftover branches from the stump and scatter them.
+* Do not cut any tree designated with paint. 
 * You cannot cut trees in the Boulder or Clear Creek Ranger Districts.
 
 ##### Sulphur Ranger District
 
 The below rules are specific to the Sulphur Ranger District:
 
 * Tree cutting is permitted from {{sulphurDate}}.
-* You can only cut trees from areas that are specifically designated for Christmas tree cutting. Be sure to reference the [map](/assets/img/sulphur-map.png) to ensure you are in a permitted cutting area. 
+* You can only cut trees from areas that are specifically designated for Christmas tree cutting. Be sure to reference the [map](/assets/img/site-wide/sulphur-map.png) to ensure you are in a permitted cutting area. 
```

Comment:
It seems like there are maps in both `img/maps/` and`img/site-wide/` i think  we might want to move all maps into `maps`

Reply:
@lauraGgit I fixed this path as part of #125, and all the maps are under `/maps` now.